kzg_utils/Cargo.toml

```diff
@@ -20,3 +21,11 @@ test-generator = { workspace = true }
 
 [lints]
 workspace = true
+
+[features]
+# `backend-any` should not be passed to Cargo.
+# It only exist to avoid duplicating lists of features.
+backend-any = []
```

Comment:
I would like something like "bls-backend-any" (I prefer this more) or "ecc-backend-any" (I prefer this less).